cli/src/opts/cast.rs

```diff
@@ -298,6 +301,17 @@ pub enum Subcommands {
         #[structopt(subcommand)]
         command: WalletSubcommands,
     },
+    #[structopt(name = "interface", about = "generate contract's interface from ABI")]
+    Interface {
```

Comment:
Add an extra param and use that in the consuming function, we should move away from the API key helper function:

```rust
#[structopt(help = "Your Etherscan API key", env = "ETHERSCAN_API_KEY")]
etherscan_api_key: String
```